documentation/modules/exploit/multi/http/motioneye_auth_rce_cve_2025_60787.md

```diff
@@ -0,0 +1,58 @@
+## Vulnerable Application
+
+This module exploits a template injection vulnerability in the MotionEye.
+
+MotionEye is vulnerable to OS Command Injection in configuration parameters such as image_file_name. 
```

Comment:
```suggestion
MotionEye is vulnerable to OS Command Injection in configuration parameters such as `image_file_name`. 
```

Reply:
Fixed in https://github.com/rapid7/metasploit-framework/pull/20585/commits/25624e9ae0cd5a456752aae337bb12825b5b4322